### PR TITLE
First changes

### PR DESCRIPTION
**Pull request by Gab0rB**
Included OS Check to run npm, grunt, bower and yo commands correctly  via script. There's a ".cmd" extension needed  to run them correctly on windows OS.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -13,6 +13,17 @@ plugins {
...
 
+//Check OS for correct usage of npm, bower and grunt commands
+
+import org.gradle.api.tasks.Exec
+import org.apache.tools.ant.taskdefs.condition.Os
+
+ext {
+	npmCommand = Os.isFamily(Os.FAMILY_WINDOWS) ? 'npm.cmd' : 'npm'
+	bowerCommand = Os.isFamily(Os.FAMILY_WINDOWS) ? 'bower.cmd' : 'bower'
+	gruntCommand  = Os.isFamily(Os.FAMILY_WINDOWS) ? 'grunt.cmd' : 'grunt'
```
Comment (mschaaf) [inline build.gradle]: nice.
Please follow the code formatting 2 spaces indentation.

**Comment by mschaaf** [comment]
looks good, only the formatting

**Comment by mschaaf** [comment]
@Gab0rB is it ok if I close it? I guess the important changes are already added, right?

**Comment by Gab0rB** [comment]
yes, they are, you can close it.

**Pull request closed without merging**